components/content-search/index.tsx

```diff
@@ -375,15 +452,12 @@ const ContentSearch = ({
 					onFocus={() => {
 						setIsFocused(true);
 					}}
-					onBlur={() => {
-						setIsFocused(false);
-					}}
 				/>
 
 				{hasSearchString || hasInitialResults ? (
 					<>
 						<List className={`${NAMESPACE}-list`}>
-							{isLoading && currentPage === 1 && <StyledSpinner />}
+							{isLoading && currentPage === 1 && <StyledSpinner onPointerEnterCapture={null} onPointerLeaveCapture={null} />}
```

Comment:
Weirdly we have to pass these two props to Spinner (specifically to SVG-based component), to stop Typescript from complaining. Otherwise, Typescript throws an error.
There is a open issue for this on the Gutenberg repo - https://github.com/WordPress/gutenberg/issues/61322

But the issue is setting these props throws warning in the console now by SVG element -

<img width="576" alt="Screenshot 2024-05-16 at 1 35 12 AM" src="https://github.com/10up/block-components/assets/39427067/5413f404-e3a9-438c-a438-efdd0c5b7263">